plugins/mimecast/unit_test/test_monitor_siem_logs.py

```diff
@@ -48,17 +50,6 @@ def test_monitor_siem_logs_raises_401(self, _mock_data):
         self.assertEqual(new_state, state_params)  # we shouldn't change the state if we encounter an error
         self.assertEqual(type(error), ApiClientException)
 
-    @patch("logging.Logger.error")
-    def test_monitor_siem_logs_stops_path_traversal(self, mock_logger, _mock_data):
-        test_state = {"next_token": "path_traversal"}  # this forces our mock util to append `../` into filenames
-        response, new_state, has_more_pages, status_code, error = self.task.run(params={}, state=test_state)
-        self.assertEqual(status_code, 200)
-        self.assertEqual(has_more_pages, True)
-        self.assertEqual(response, [])  # no logs will be parsed as we raise error after catching BadZipFile
-        self.assertEqual(new_state, test_state)  # we shouldn't change the state if we encounter an error
-        mock_logger.assert_called()
-        self.assertIn("Hit BadZipFile, returning []. Error", mock_logger.call_args[0][0])
-
```

Comment:
Why pull this test?

Reply:
Pulled this as I added in a sanitise method in 5.3.3 - which may be the culprit of the negative seek file. so this test now fails